ext/http/Cargo.toml

```
@@ -2,7 +2,7 @@
 
 [package]
 name = "deno_http"
-version = "0.33.0"
+version = "0.34.0"
```

```suggestion
version = "0.31.0"
```
Do you think we can use the old version without bumping?

Release a new version would create a situation where we have the compression feature in `0.32.0/0.33.0` but not in `0.34.0`. Since we don't have any changes, we probably can use `0.31.0`?

---

Ah, we published 0.32.0 and 0.33.0 independent of CLI release process?

Ok. Let's try that